src/features/retreat/components/islandTravel/IslandTravelWrapper.tsx

```diff
@@ -0,0 +1,13 @@
+import { useActor } from "@xstate/react";
+import { IslandTravel } from "features/game/expansion/components/IslandTravel";
+import { Context } from "features/game/GoblinProvider";
+import React, { useContext } from "react";
+
+export const IslandTravelWrapper = () => {
```

Comment:
What is this component wrapping?

Reply:
This wrapper is basically to access the context. Our component IslandTravel requires bumpkin as a prop. The reason we were not able to place IslandTravel directly in Retreat.tsx is that the GoblinProvider is initialised in that component. So we cannot access the context in Retreat.tsx itself. So we had to make a wrapper to access the context.